web/index.html

```
@@ -70,6 +75,92 @@ <h1>Mappa</h1>
+                transition: background 0.2s;
+            }
+            button:hover:not(:disabled) { background: var(--accent-hover, #e94560); }
+        </style>
+        <div class="search-row">
+            <div class="combobox" role="combobox" aria-expanded="false" aria-haspopup="listbox" aria-owns="listbox">
+                <input type="text" id="input" placeholder="Search npm packages..."
+                       autocomplete="off" aria-autocomplete="list" aria-controls="listbox"
+                       aria-label="Search npm packages">
+                <ul id="listbox" role="listbox" hidden></ul>
+            </div>
+            <select id="version" disabled aria-label="Package version">
```

HTMLHint (1.9.2)

[error] The id value [ version ] must be unique.

(id-unique)